packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/data_sources/remote/auth_data_source.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021, Prime Holding JSC
+// https://www.primeholding.com
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:dio/dio.dart';
+import 'package:retrofit/http.dart';
+
+import '../../models/auth_token_model.dart';
+import '../../models/request_models/authenticate_user_request_model.dart';
+
+part 'auth_data_source.g.dart';
+
+@RestApi(baseUrl: 'http://0.0.0.0:8080')
+abstract class AuthDataSource {
+  factory AuthDataSource(Dio dio, {String baseUrl}) = _AuthDataSource;
```

Comment:
Not sure about this one, as the url specified here is never used, since we're assigning a new one inside the app_dependencies. This address is treated as a fallback address in case when none is specified. Also, (potentially) using different data sources may result in different base addresses.